px-data-grid.html

```diff
@@ -840,8 +775,8 @@
           }
 
           // TODO: these values should be in params!
-          if (this.filters && this.filters.length != 0) {
-            items = this._applyCustomFilter(items, this.filters);
+          if (this._filters && this._filters.length != 0) {
```

Comment:
Notice that also px-data-grid-pagination is accessing this variable, as it's not provided in params. Also it will not be at all available for 3rd party data providers now.